floyd/cli/experiment.py

```diff
@@ -164,26 +165,27 @@ def delete(ids, yes):
     """
     Delete project runs
     """
-    
+    failures = False
     for id in ids:
         experiment = ExperimentClient().get(id)
-        task_instance_id = get_module_task_instance_id(experiment.task_instances)
-        task_instance = TaskInstanceClient().get(task_instance_id) if task_instance_id else None
-
+        if not experiment:
+            failures = True
+            continue
 
-        if experiment.state in ["queued", "running"]:
```

Comment:
Agreed, I'm working on the backend to delete the module as part of the delete experiment operation. After I merge that change, I will remove the module delete step from here.

In the meantime, I like the approach of deleting the experiment first.